test/powershell/Modules/Microsoft.PowerShell.Utility/assets/docker/ClientCertificateCheck/Pages/_Layout.cshtml

```diff
@@ -0,0 +1 @@
+@RenderBody()
```

Comment:
@iSazonov  `_Layout.cshtml` and `index.cshtml` intentional as those determine the response stream and superfluous new lines cause it to response less like a raw replay and more like an HTML page. I will correct the others.